chrome/content/zotero/elements/itemPaneSidenav.js

```diff
@@ -210,13 +215,13 @@
 			}
 
 			this.addEventListener('click', this.handleButtonClick);
-			
+			this.addEventListener('keydown', this.handleKeyDown);
 			// Set up action toolbarbuttons
 			for (let toolbarbutton of this.querySelectorAll('toolbarbutton[data-action]')) {
 				let action = toolbarbutton.dataset.action;
 				
 				if (action === 'locate') {
-					toolbarbutton.addEventListener('mousedown', async (event) => {
+					toolbarbutton.addEventListener('click', async (event) => {
```

Comment:
The issue I ran into with `mousedown` was that a click via `space` or `enter` on a `toolbarbutton` would not trigger the `mousedown` listener. But a `click` listener would run just fine.

I just reverted this back to  `toolbarbutton.addEventListener('mousedown', ...` because now in `zoteroPane.js`, on `space` or `enter` we call `event.target.click()` which triggers this listener as well.